src/components/HomePage/SearchArea/SubmitSearchButton.spec.js

```diff
@@ -11,8 +11,16 @@ describe("SubmitSearchButton", () => {
                     .find(Fab).exists()
             ).toBe(true);
         });
-        it("should render 'search' icon", () => {
-            expect(shallow(<SubmitSearchButton />).find(Search).exists()).toBe(true);
+        it("should render 'search' icon when searchHasInput = true", () => {
+            expect(shallow(<SubmitSearchButton searchHasUserInput={true} />).find(Search).exists()).toBe(true);
+        });
+        it("should render 'span' with text = 'Show All' when searchHasInput = false", () => {
+            const button = shallow(
+                <SubmitSearchButton
+                    searchHasUserInput={false}
+                />
+            );
+            expect(button.find("span").first().text()).toEqual("Show All");
         });
     });
```

Comment:
These tests work but if you could rewrite them using the functions from React Testing Library (i believe they are the `render` and `renderWith*` variants) it would be great.

`shallow` is a function given by the `enzyme` module which is no longer maintained and tests using methods from Enzyme should migrate to other tools: in our case that is RTL. @imnotteixeira had some work migrating most of the tests from Enzyme to RTL so if there are any questions you can ask him but I believe this is an easy fix.

Reply:
> if you could rewrite them using the functions from React Testing Library it would be great.

I'd go further: we should have no new code with enzyme tests whatsoever. We had a lot of work migrating existing tests to RTL, and we should be using it, and trying to migrate the last few so we can finally remove the dependency on Enzyme.

Please do not merge PRs with Enzyme tests

Reply:
Should we also change the `config/test.js` file created by the CRA `eject` command? The Enzyme globals are defines there.

Reply:
Once enzyme is not used, yes